src/api/zodClient.ts

```diff
@@ -756,6 +756,20 @@ const ProjectUserProfile = z.object({
 const UpdateProjectUserProfileDto = z.object({
     dvcUserId: z.string().nullable(),
 })
+const Override = z.object({
+    _project: z.string(),
+    _environment: z.string(),
+    _feature: z.string(),
+    _variation: z.string(),
+    dvcUserId: z.string(),
```

Comment:
does this support empty strings, null and undefined?

Reply:
not sure, but we would only get the response for the UpdateCommand if all values are set and this is the return object